app/models/bars/BarsCommon.scala

```diff
@@ -0,0 +1,200 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.bars
+
+import cats.Eq
+import essttp.rootmodel.bank.{AccountNumber, SortCode}
+import org.apache.commons.lang3.StringUtils
+import play.api.libs.json.{Format, Json}
+
+object BarsCommon {
+
+  // request
+
```

Comment:
Maybe overkill, but you could wrap this in a BarsRequest object or something, then it documents itself, same for response below

Reply:
Actually ignore this